pyproject.toml

```diff
@@ -37,6 +38,9 @@ dependencies = [
 ]
 dynamic = ["version"]
 
+[project.scripts]
+trading-cli = "frequenz.client.electricity_trading.__main__:main"
```

Comment:
should this file be located in `src/frequenz/client/electricity_trading/cli/__main__.py` instead?